lang/aladino/report.go

```diff
@@ -90,8 +90,12 @@ func BuildVerboseReport(report *Report) string {
 	sb.WriteString("```yaml\n")
 
 	// Report
-	for _, action := range report.Actions {
-		sb.WriteString(fmt.Sprintf("%v\n", action))
+	if len(report.Actions) == 0 {
+		sb.WriteString("No actions executed\n")
+	} else {
```

Comment:
We can return immediately here and allow for better reading.

```suggestion
	if len(program.GetProgramStatements()) == 0 {
		sb.WriteString("No actions triggered\n```\n")
		return sb.String()
	}
```

Reply:
Hmm, how will this allow for better reading, @ferreiratiago?
We are only handling the case of whether we have actions executed or not, after that the treatment is exactly the same.
I don't see the need to repeat the code unnecessarily 🤔

Reply:
It's the idea of [line of sight](https://speakerdeck.com/fedepaol/reducing-cognitive-load-yet-another-idiomatic-go-talk?slide=20) in order to reduce the cognitive load. In this case in particular is not a big deal but I think we should take this idea into account when making changes. You can check Federico [talk](https://fosdem.org/2023/schedule/event/goreducecognitive/) here.

Reply:
@ferreiratiago Yes, I am aware of the concept precept. However, being this such a simple case, adding more code may do the exact opposite and cause noise.